src/style.css

```diff
@@ -1,3 +1,115 @@
+/* Configuração geral da fonte */
 body {
-  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+/* Estilização do cabeçalho */
+header {
+  display: flex;
+  align-items: center;
+  width: 1024px;
+  padding: 0 20px;
+  background-color: white;
+  height: 60px;
+  position: relative;
+  margin: 0 auto;
+  justify-content: space-between;
+}
+
+/* Estilização da imagem do logo */
+.logo img {
+  width: 40px;
+  height: 40px;
+  position: relative; /* Para que as propriedades top e left tenham efeito */
+  top: 10px; /* Ajuste a posição do logo conforme necessário */
+  left: 50px; /* Ajuste a posição do logo conforme necessário */
+}
+
+/* Estilização da navegação */
+nav ul {
+  list-style: none;
+  display: flex;
+  align-items: center;
+}
+
+/* Estilização dos itens da lista */
+nav ul li {
+  margin-right: 20px; /* Espaçamento entre os links */
+  position: relative;
+}
+
+/* Estilização dos links */
+nav ul li a {
+  text-decoration: none;
+  font-weight: 700;
+  color: black;
+  position: relative; /* Permite que os filhos usem position absolute em relação a ele */
+  line-height: 14.06px;
+  text-align: center;
+  transition: color 0.3s ease;
+  font-size: 12px;
+}
```

Comment:
There's a repetition of styles for `.nav ul li a` and each of its subclasses (`.apple`, `.samsung`, etc.). You can remove the font-weight and font-size declarations from the subclasses since they are already defined in the `.nav ul li a` class.